kapitan/cli.py

```diff
@@ -270,6 +270,23 @@ def build_parser():
         help='set schema cache path, default is "./schemas"',
     )
 
+    compile_parser.add_argument(
+        "--multiline-string-style",
```

Comment:
can we rename this to `--yaml-multiline-string-style`?

Reply:
Of course!